From 2510e21d2cd60429ff30ef150e409e010371547a Mon Sep 17 00:00:00 2001
From: Jesse Young <jlyo@jlyo.org>
Date: Mon, 7 Sep 2015 09:17:47 -0500
Subject: [PATCH 3/4] init: make mdadm quiet

---
 initramfs-init.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/initramfs-init.in b/initramfs-init.in
index d687e64..78500aa 100755
--- a/initramfs-init.in
+++ b/initramfs-init.in
@@ -227,12 +227,14 @@ setup_inittab_console(){
 
 start_raid() {
 	local n= i=
+	local mdadm_flags=
 	case "$KOPT_root" in
 		/dev/md*) n=${KOPT_root#/dev/md} ;;
 	esac
 	case "$KOPT_autoraid" in
 		[0-9]*) n="$n $(echo $KOPT_autoraid | tr ',' ' ')" ;;
 	esac
+	[ "$KOPT_quiet" = yes ] && mdadm_flags="$mdadm_flags -q"
 	# if kernel can autostart the raid he will
 	for i in $n; do
 		mknod /dev/md$i b 9 $i
@@ -241,7 +243,7 @@ start_raid() {
 	# kernel cannot autostart newer versions of mdadm metadata
 	# so we also check if mdadm binary is there
 	if [ -x /sbin/mdadm ]; then
-		mdadm --assemble --scan
+		mdadm $mdadm_flags --assemble --scan
 	fi
 }
 
-- 
2.4.1

